docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -804,9 +804,8 @@ ha_tracker:
   # CLI flag: -distributor.ha-tracker.failover-timeout
   [ha_tracker_failover_timeout: <duration> | default = 30s]
 
-  # Backend storage to use for the ring. Please be aware that memberlist is not
-  # supported by the HA tracker since gossip propagation is too slow for HA
-  # purposes.
+  # Backend storage to use for the ring. Please be aware that memberlist is
+  # supported by the HA tracker but its experimental.
```

Comment:
```suggestion
  # Backend storage to use for the ring. Note that memberlist
  # support by the HA tracker is experimental.
```

Reply:
We avoid using "Please" in docs. Also, I think this sounds a bit more definitive.

Reply:
Thank you @tacole02 . Addressed here [94e86e8](https://github.com/grafana/mimir/pull/10054/commits/94e86e86730024d4fbf3fab2c8d47f15cd470bfc)